### PR TITLE
live: add to v8 upgrading section

### PR DESCRIPTION
**Pull request by FZambia**
**What this PR does / why we need it**:

Since some users come across problems during upgrade to v8 we should document Live caveats in upgrading guide.

**Which issue(s) this PR fixes**:

Relates #36614, also https://github.com/grafana/grafana/issues/35448#issuecomment-872224175

### CODE REVIEW

**Review by marefr — APPROVED** [review]
LGTM, but might want to wait for review from technical writer

File: docs/sources/installation/upgrading.md
Comment (achatterjee-grafana) [inline docs/sources/installation/upgrading.md:334]:
```suggestion
When WebSocket connection is established, Grafana checks the request Origin header due to security reasons (for example, to prevent hijacking of WebSocket connection). If you have a properly defined public URL (`root_url` server option) then the origin check should successfully pass for WebSocket requests originating from public URL pages. In case of an unsuccessful origin check, Grafana returns a 403 error. It's also possible to add a list of additional origin patterns for the origin check.
```

File: docs/sources/installation/upgrading.md
Comment (achatterjee-grafana) [inline docs/sources/installation/upgrading.md:336]:
```suggestion
To handle many concurrent WebSocket connections you may need to tune your OS settings or infrastructure. Grafana Live is enabled by default and supports 100 concurrent WebSocket connections max to avoid possible problems with the file descriptor OS limit. As soon as your setup meets the requirements to scale the number of persistent connections this limit can be increased. You also have an option to disable Grafana Live.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

File: docs/sources/installation/upgrading.md
Comment (achatterjee-grafana) [inline docs/sources/installation/upgrading.md:338]:
```suggestion
Refer to [Grafana Live configuration]({{< relref "../live/configure-grafana-live.md" >}}) documentation for more information.
```

**Review by achatterjee-grafana — COMMENTED** [review]
One more.

**Pull request merged**
